spark/src/test/java/io/ddf/spark/TableNameReplacerTests.java

```diff
@@ -447,29 +429,35 @@ public void BatchTestRelationalOps() {
 
     @Test
     public void testBlankQuotation() {
-        TableNameReplacer tableNameReplacer  = new TableNameReplacer(manager);
-
         String sqlcmd = "select value from a where value = \"\"";
         Statement statement = null;
         try {
             statement = parser.parse(new StringReader(sqlcmd));
         } catch (JSQLParserException e) {
-            e.printStackTrace();
             assert(false);
         }
     }
 
+    @Test
+    public void testSelectOne() {
+      String sqlcmd = "select 1";
+      try {
+        tableNameReplacer.run(parser.parse(new StringReader(sqlcmd)));
+      } catch (Exception e) {
+        assert false;
+      }
+    }
 
     @BeforeClass
     public static void startServer() throws Exception {
```

Comment:
Agree, but this is white line, not in this PR scope :D let's do it while refactoring the tests